e2e/test/scenarios/dashboard-filters/temporal-unit-parameters.cy.spec.js

```diff
@@ -0,0 +1,49 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import {
```

Comment:
_This might be an existing issue_

I have a question [with 3 temporal breakouts](https://github.com/metabase/metabase/assets/6830683/bfec7e48-aada-470f-a556-715ce755e71a) and I can't differentiate between them when trying to connect a dashboard filter:

![image](https://github.com/metabase/metabase/assets/6830683/3ee49fc0-f304-4bb8-99f5-57e042b39326)

Reply:
Looks like the menu uses `displayName` instead of `longDisplayName`